docs/src/test/java/docs/home/persistence/CqrsIntegrationTest.java

```diff
@@ -98,20 +98,18 @@ private ReadSide readSide() {
 
   //yeah, the Akka testkit is in need of some Java 8 love
   private void eventually(Effect block) {
-    new JavaTestKit(system) {
+    new TestKit(system) {
       {
-        new AwaitAssert(Duration.create(20, TimeUnit.SECONDS)) {
-          @Override
-          protected void check() {
-            try {
-              block.apply();
-            } catch (RuntimeException e) {
-              throw e;
-            } catch (Exception e) {
-              throw new RuntimeException(e);
-            }
+        awaitAssert(duration("20 seconds"), () -> {
```

Comment:
Just wanted to mention that I'm struggling with `java.time.Duration` vs `scala.concurrent.duration.Duration`

As per latest doc/code from `akka-testkit` there is no such method : 

```
public <A> A awaitAssert(java.time.Duration max,
                         java.util.function.Supplier<A> a)
```
however the docs suggests to use it : 

https://doc.akka.io/japi/akka/current/akka/testkit/javadsl/TestKit.html#awaitAssert-scala.concurrent.duration.Duration-java.util.function.Supplier-

Reply:
To tackle this one I opened https://github.com/akka/akka/pull/25052